vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,107 @@
+# Reconcile Mouse APIs in React Native - Desktop
+
+Windows and macOS support mouse devices today, but their React Native counterparts are not fully flushed out yet to unblock customers attempting to create React Native apps on those devices.
+
+This document will attempt to capture what's needed within React Native for Windows and macOS to enable customer scenarios. This includes:
+
+- Aligning with the [react-native-web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0) mouse behavior/implementation [issue TBD]()
+- Ensuring mouse behavior is consistent across the two platforms we support
+- Once our two desktop platforms are aligned with React Native Web, we upstream this implementation to core [issue TBD]()
+
+## `hovered`
+Property on `Pressable` that determines whether or not the mouse is currently within the bounds of the designated content.
```

Comment:
To make sure we specify the right language, `hovered` isn't itself a property on `Pressable` like `onMouseEnter`/`onMouseLeave`, but is now provided to callbacks on `Pressable` determining style/children. https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0#diff-050d4b1a0df82333a1d0ee015e944087f690fe688ad0aea9632977981284a182R23-R27

Reply:
I see, does that mean we'll be introducing onPointerEnter/onPointerLeave on Pressable instead?